test/e2e/tests/settings/address-book.spec.js

```diff
@@ -70,6 +70,56 @@ describe('Address Book', function () {
       },
     );
   });
+
+  it('Adds a new contact to the address book', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+        await openMenuSafe(driver);
+
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.clickElement({ text: 'Contacts', tag: 'div' });
+
+        await driver.clickElement('.address-book__link');
+
+        const inputUsername = await driver.findElement('#nickname');
+        await inputUsername.fill('Test User');
+
+        const inputAddress = await driver.findElement(
+          'input[placeholder="Enter public address (0x) or domain name"]',
+        );
+        await inputAddress.fill('0x56A355d3427bC2B1E22c78197AF091230919Cc2A');
```

Comment:
This is similar cause race condition and should avoid using input and replace with stronger locator

`await driver.fill('[data-testid="ens-input"]','0x56A355d3427bC2B1E22c78197AF091230919Cc2A');`